### PR TITLE
Short Answer Questions

### PR DESCRIPTION
**Pull request by markrenaud**
Added README.md containing answers to Part 1 (short answer) and Part 3 (above and beyond).

### CODE REVIEW

File: Module1 Homework/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+Refer to the code in `ContentView.swift`.
+
+
+## Part 3 - Above and Beyond
+
+> These answers refer to code in the Homework document.
+
+In ContentView, lines 1 and 2 show the definition of `State Properties`.
+
+In ContentView, line 3 shows the definition of an `Instance Property`.
```
Comment (niknovak) [inline Module1 Homework/README.md:172]: Instance is okay, however we're looking here more towards stored property. This is the important difference between lines 3 and 4. One is a stored property while the other is computed.

File: Module1 Homework/README.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+In ContentView, line 4 shows the definition of a `Computed Property`.
+
+In ContentView, line 5 shows an `Instance` of Game calling the `Method` points.
+
+In ContentView, line 6 is the definition of the `Method` doSomethingWithCounter().
+
+In Game, lines a, b, and c show the definition of `Instance Properties`.
```
Comment (niknovak) [inline Module1 Homework/README.md:180]: Just as above, stored properties would be more complete answer.

File: Module1 Homework/README.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+In Game, line d is the definition of the `Function` points(sliderValue: Int).
+
+Lines 3, a, b, and c are the `Instance Properties` and lines 6 and d are the `Methods` of the structs.
```
Comment (niknovak) [inline Module1 Homework/README.md:184]: Stored as well :]

File: Module1 Homework/README.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+In ContentView, line 4 shows the definition of a `Computed Property`.
+
+In ContentView, line 5 shows an `Instance` of Game calling the `Method` points.
+
+In ContentView, line 6 is the definition of the `Method` doSomethingWithCounter().
+
+In Game, lines a, b, and c show the definition of `Instance Properties`.
+
+In Game, line d is the definition of the `Function` points(sliderValue: Int).
```
Comment (niknovak) [inline Module1 Homework/README.md:182]: A method would be more precise. Functions are independent while methods belong to some data type.

File: Module1 Homework/README.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+- Create multiple previews using the `#Preview` macro in Xcode 15:
+```
+#Preview("Portrait") {
+    ContentView()
+        .previewInterfaceOrientation(.portrait)
+}
+#Preview("Landscape Left") {
+    ContentView()
+        .previewInterfaceOrientation(.landscapeLeft)
+}
```
Comment (niknovak) [inline Module1 Homework/README.md:42]: This won't work anymore in Xcode 15.
We can define the orientation like this:

```swift
#Preview(traits: .landscapeLeft) {
    ContentView()
}
```

We're going to need this in one of the next modules. :]

File: Module1 Homework/README.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+    }
+}
+```
+
+#### _g) An app is made up of **`instances`** of **`classes`** and **`structs`** that contain **`data`** and **`methods`**._
+
+#### _h) Name two components of a SwiftUI Button._
+
+- A `title` to display.
```
Comment (niknovak) [inline Module1 Homework/README.md:64]: A `Label` would be more precise here, which can have a title and an icon.

**Review by niknovak — APPROVED** [review]

File: Module1 Homework/README.md (hunk 1 above)
Comment (markrenaud) [reply Module1 Homework/README.md:172]: True.  Stored property is much more Swifty sounding (especially for a struct).

File: Module1 Homework/README.md (hunk 4 above)
Comment (markrenaud) [reply Module1 Homework/README.md:182]: So true.

File: Module1 Homework/README.md (hunk 5 above)
Comment (markrenaud) [reply Module1 Homework/README.md:42]: Interestingly, `.previewInterfaceOrientation(.landscapeLeft)` still works for me in Xcode 15 with `#Preview` macro.  However, I had missed the whole `traits` parameter on the `#Preview` macro - and obviously the intended way to proceed moving forward.  Learned something there.  Thanks.

**Pull request merged**